src/components/Mission.component.jsx

```diff
@@ -15,6 +15,7 @@ function Mission({ mission }) {
 
   const joinHandler = (e) => {
     const { id } = e.target;
+    console.log(id);
```

Comment:
According to JS best practice, kindly remove all console.log in your code to make it clean"

Reply:
Thanks @jodavid444 ,
I already fixed it. Please review it again.